modules/vlada-m/src/test/java/com/hillel/elementary/javageeks/task5/TextFormatterTest.java

```diff
@@ -0,0 +1,16 @@
+package com.hillel.elementary.javageeks.task5;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TextFormatterTest {
+
+    @Test
+    void shouldFormatText() {
+        String text = "One night—it was on the 20th of March, 1888—I was returning from a journey to a patient (for I had now returned to civil practice), when my way led me through Baker Street. As I passed the well-remembered door, which must always be associated in my mind with my wooing, and with the dark incidents of the Study in Scarlet, I was seized with a keen desire to see Holmes again, and to know how he was employing his extraordinary powers.";
```

Comment:
let's add at least one work which starts with alphabetic symbol and has non-alphabetic symbol in in to the test case